### PR TITLE
Whatsnew for users & sysadmins & CLI signposting

### PR DESCRIPTION
**Pull request by hflynn**
See https://trello.com/c/UloMh1fl/108-document-new-features-in-whats-new-pages-and-on-website

This adds content for Users and Sysadmins and also highlights the CLI docs from the top index page and the User index page ( @rleigh-codelibre pointed this issue out - the change to the 'About the OMERO Platform' heading dated from all the user docs being moved to help but now we have CLI docs here that are basically hidden unless you read everything really thoroughly!)

Still to do

- [x] What's new for developers for 5.4.0 See https://github.com/openmicroscopy/ome-documentation/pull/1767
- [x] Decide whether we are renaming the secvuln extlink or changing it to use the new website URL rather than /info/vulnerabilities  See https://github.com/openmicroscopy/ome-documentation/pull/1769

### CODE REVIEW

File: omero/users/clients-overview.rst [hunk 1]
```diff
@@ -155,3 +155,11 @@ for a detailed list of supported formats.
     
     OMERO.importer
 
+OMERO.cli
+---------
+
+The |CLI| is a set of Python_ based system-administration, deployment and
```
Comment (mtbc) [inline omero/users/clients-overview.rst:161]: I'd be inclined to move the hyphen from "system administration" to "Python-based", just a suggestion though.

File: omero/users/whatsnew.rst [hunk 2]
```diff
@@ -1,35 +1,24 @@
...
-  OMERO.figure or other plugins directly from OMERO.web (coming soon is a new
-  web-based full image viewer - OMERO.iviewer)
-- ability to filter by ratings in OMERO.web
-- updated Bio-Formats bringing improved file format support including support
-  for JPEG-XR compressed CZI data
+- Your system administrator can now create
+  :doc:`/sysadmins/admins-with-restricted-privileges` to allow facility
+  managers, image analysts etc. to organize users and data in OMERO
+  without having to be granted full administrator privileges. This is a way to
+  cater for the needs of some trusted users to act on behalf of all the OMERO
```
Comment (mtbc) [inline omero/users/whatsnew.rst:10]: maybe "need for" instead of "needs of"

File: omero/users/whatsnew.rst [hunk 3]
```diff
@@ -1,35 +1,24 @@
...
+  users in a facility, with access to all the groups and data in OMERO.
+
+- Improvements to OMERO.web including the display of plates and wells, the
+  synchronization of controls enabling copy/paste of rendering settings, and
+  the layout of the left-hand panel.
+
+- The 'Reverse Intensity' command in the image viewers has been renamed to
+  'Invert'.
+
+- updated Bio-Formats bringing improved file format support.
```
Comment (mtbc) [inline omero/users/whatsnew.rst:20]: need initial capital?

**Comment by jburel** [comment]
Closing this PR since @hflynn is away 
I will re-open

**Comment by pwalczysko** [comment]
Sorry if I am commenting on a wrong PR, but both here as well as on https://github.com/openmicroscopy/ome-documentation/pull/1765 I think I am missing a mention of a new feature in OMERO.web, where Full Admin or Administrator with Restricted Privileges with correct privileges ("Write data") can create P/D/S for others when in single-user view, as described in https://github.com/openmicroscopy/ome-help/pull/251/files#diff-cda682c7f7f039024b036b6a3c4186daR990

Also, there are some improvements on the CLI (chown all belonging to user, see https://github.com/openmicroscopy/ome-documentation/pull/1727/files#diff-b2d2c814858d6db577d2ea5539660db5R43 - note that we probably have to mention that this feature is advanced and might be potentially slow.

Both of these missing features metnioned here are for sysadmins or light admins. It really depends how we define "user" (simple user or superuser) to decide in which What's new they should be mentioned. I would suggest to mention them in both Whats's news (admins and users)

cc @hflynn @jburel

**Pull request closed without merging**